.changeset/chilly-results-relax.md

```diff
@@ -0,0 +1,6 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+
+Optional schema validation on newly inserted content
```

Comment:
Nit (since these end up in user-facing documentation)
```suggestion
Added support for optional schema validation on newly inserted content in SharedTree.
```

Reply:
I originally added this more for the sake of having the changeset in place, and planning to review the content later, but this makes sense and I think I have a decent idea for the detailed change description, updating.